cime_config/allactive/config_pesall.xml

```diff
@@ -6958,22 +6958,22 @@
         </nthrds>
         <rootpe>
           <rootpe_atm>0</rootpe_atm>
-          <rootpe_lnd>540</rootpe_lnd>
-          <rootpe_rof>540</rootpe_rof>
+          <rootpe_lnd>648</rootpe_lnd>
+          <rootpe_rof>648</rootpe_rof>
           <rootpe_ice>0</rootpe_ice>
           <rootpe_ocn>684</rootpe_ocn>
           <rootpe_cpl>0</rootpe_cpl>
         </rootpe>
       </pes>
       <pes compset=".*EAM.+ELM.+MPASSI.+MPASO.+MOSART.+" pesize="M">
-        <comment> --compset WCYCL* --res ne30pg2_EC30to60E2r2 on 48 nodes pure-MPI, ~8.8 sypd </comment>
+        <comment> --compset WCYCL* --res ne30pg2_EC30to60E2r2 on 48 nodes pure-MPI, ~9.4 sypd </comment>
         <ntasks>
           <ntasks_atm>1350</ntasks_atm>
-          <ntasks_lnd>288</ntasks_lnd>
-          <ntasks_rof>288</ntasks_rof>
-          <ntasks_ice>1080</ntasks_ice>
+          <ntasks_lnd>72</ntasks_lnd>
+          <ntasks_rof>72</ntasks_rof>
```

Comment:
Maybe adjust these from 72 to 144? GNU is running into OOM for
```
SMS_Ld2.ne30pg2_r05_EC30to60E2r2.BGCEXP_CNTL_CNPECACNT_1850.anvil_gnu.elm-bgcexp
SMS_Ld2.ne30pg2_r05_EC30to60E2r2.BGCEXP_CNTL_CNPRDCTC_1850.anvil_gnu.elm-bgcexp
```

Reply:
@amametjanov - what do you think about modifying a new default based on either the ne30pg2_r05_EC30to60E2r2 resolution or the BGC compsets? Or maybe a combination of the two? I'm happy to take that on, but I'd appreciate understanding your preference

Reply:
This layout is for `ne30pg2_EC30to60E2r2` + `WCYCL1850` cases and it also gets picked up for `ne30pg2_r05_EC30to60E2r2` + `BGCEXP_CNTL_CNPECACNT_1850` test-cases.

As you suggest, it's likely better to add a new small node-count default for BGC compsets (and keep this WCYCL layout). Please take a look at #4886, which disambiguates WCYCL (long compset name ends with `SESP` [with EOL regex `$`]) and BGC (long compset name contains `BGC` after `SESP`). It should run on the fewest number of nodes without running into OOM with either intel or gnu compilers.